From b59f9a7ecbe26c1dd00db1200e560f67abcfa8ea Mon Sep 17 00:00:00 2001
From: tropf <tropf@noreply.codeberg.org>
Date: Tue, 8 Aug 2023 22:10:25 +0200
Subject: [PATCH 6/9] electron: fix route detection

The nix electron invocation starts electron as
`electron main.js PATH`. However, Inkstitch expects invocation as
`electron-binary PATH`, and subsequently uses (hardcoded) argv[1] for
routing. This is changed to use argv[2] to conform to the nix
invocation.
---
 electron/src/main/index.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/electron/src/main/index.js b/electron/src/main/index.js
index 3158007d..f014c961 100644
--- a/electron/src/main/index.js
+++ b/electron/src/main/index.js
@@ -25,7 +25,7 @@ var target = null
 if (process.argv.includes(printPdfUrl)) {
     target = printPdfUrl
 } else {
-    target = process.argv[1] || "";
+    target = process.argv[2] || "";
 }
 var targetURL = url.parse(target)
 var winURL = null
-- 
2.40.1

